pom.xml

```diff
@@ -52,6 +52,11 @@
       </dependency>
   </dependencies>
 
+  <properties>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
```

Comment:
To have findbugs work in non-English locales.
(Though, unfortunately, doesn't work for findbugs-maven-plugin 3.0.1 for https://github.com/gleclaire/findbugs-maven-plugin/issues/18 .)